src/System.Private.CoreLib/shared/System/DateTime.cs

```diff
@@ -90,6 +90,9 @@ namespace System
         internal const long MaxTicks = DaysTo10000 * TicksPerDay - 1;
         private const long MaxMillis = (long)DaysTo10000 * MillisPerDay;
 
+        internal const int MinYear = 1;
+        internal const int MaxYear = 9999;
```

Comment:
> internal const int MaxYear = 9999; [](start = 7, length = 35)

This is already defined in [https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/Globalization/GregorianCalendar.cs#L26](https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/Globalization/GregorianCalendar.cs#L26)

I would suggest to either define  MinYear there too or let the Gregorian calendar use the ones defined here. I mean at the end we need to have these constants defined in one place